src/adapters/customVariables/ifAppInfo.ts

```diff
@@ -0,0 +1,56 @@
+import { VariableType } from '../../debugProtocol/events/responses/VariablesResponse';
+import type { DebugProtocolAdapter, EvaluateContainer } from '../DebugProtocolAdapter';
+import { pushCustomVariableToContainer } from './utils';
+
+// https://developer.roku.com/docs/references/brightscript/interfaces/ifappinfo.md
+export function pushIfAppInfoVariables(adapter: DebugProtocolAdapter, expression: string, container: EvaluateContainer) {
+    pushCustomVariableToContainer(container, {
+        name: '$id',
+        type: VariableType.String,
+        presentationHint: 'virtual',
+        evaluateName: `${expression}.GetID()`,
+        lazy: true,
+        value: '',
+        children: []
+    });
+
+    pushCustomVariableToContainer(container, {
+        name: '$dev',
```

Comment:
let's stick with the `is` `was` `has` style boolean naming from Roku. so `$isDev`. Should apply to this entire PR, just commenting on one item here.

Reply:
Has been addressed